src/es/2017-03/01/info.yml

```diff
@@ -0,0 +1,4 @@
+---
+  title: "Pablo: Apóstol a los gentiles"
+  start_date: "06/24/2017"
```

Comment:
Date format across the project should be `DD/MM/YYYY`. Here it is inconsistent with `.md` files